remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties

```diff
@@ -67,18 +67,19 @@ http-connector.security=Configuration of security for this connector.
 http-connector.property=Properties to further configure the connector.
 
 remoting.outbound-connection=Remoting outbound connections.
+remoting.outbound-connection.deprecated=Use the remote-outbound-connection resource instead, with an appropriate protocol.
```

Comment:
This doesn't do anything by itself. GenericOutboundConnectionResourceDefinition needs to be modified to record the fact it's deprecated, which is what triggers use of this text. Can shift to the constructor variant that uses a Parameters and then the fact of deprecation is set up in the parameters. I'll push something up to illustrate.

This doesn't need to block this PR but if the intent is to deprecate it needs follow up.